t/issues/t4482-flush-list-corruption.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh -e
+
+# How this test works
+#
+# use rc1 script that does not load content-sqlite
+#
+# add some unique data to the kvs, we do multiple puts to build up
+# a decent length internal list of flushable cache entries.
+#
+# drop kvs cache, this will force future KVS puts of identical data to
+# be sent to the content-cache
+#
+# write the same data again, if error present internal flush list will
+# be messed up and length of flush list < number of dirty entries
+# (acct_dirty).
+#
+# before fix, flux content flush will hang b/c number of dirty entries
+# (acct_dirty) never reaches zero.
+
+cat <<-EOF >t4482.sh
+#!/bin/sh -e
+
+flux kvs put issue4482A.a="abcdefghijk"
+flux kvs put issue4482A.b="lmnopqrstuv"
+flux kvs put issue4482A.c="wxyz0123456"
+flux kvs put issue4482A.d="7890ABCDEFG"
+flux kvs put issue4482A.e="HIJKLMNOPQR"
+flux kvs put issue4482A.f="STUVWXYZ!!!"
+flux kvs put issue4482A.g="<<<<<:>>>>>"
+
+flux kvs dropcache
+
+flux kvs put issue4482B.a="abcdefghijk"
+flux kvs put issue4482B.b="lmnopqrstuv"
+flux kvs put issue4482B.c="wxyz0123456"
+flux kvs put issue4482B.d="7890ABCDEFG"
+flux kvs put issue4482B.e="HIJKLMNOPQR"
+flux kvs put issue4482B.f="STUVWXYZ!!!"
+flux kvs put issue4482B.g="<<<<<:>>>>>"
+
+flux module load content-sqlite
+
+flux content flush
+
+EOF
+
+chmod +x t4482.sh
+
+flux start -s 1 \
```

Comment:
Since the test script is generated in place, it might be easier (and keep all test components together) to generate the custom `rc1` and `rc3` scripts here as well. I'd also hate to proliferate the one-off rc scripts in `t/rc` and end up with many `rc1-issue*` files in there in the future.

Reply:
Or, you could do away with the rc scripts and just load and unload necessary modules directly in the test script as you are doing with `content-sqlite`

Reply:
My vote would be to make the test self-contained as well.  It should probably drop the bash -e option and handle errors explicitly then, so that it doesn't bail out leaving modules unloaded for the next test.